### DISCUSSION TITLE
Developer advocates - do we see them in Research Software? (Session 3 - Room 1)

### DISCUSSION THREAD

**Discussion by rowlandm** [RSEAA/RSEAA.github.io#125] [Discussion topics for #RSEAA24]
I was talking to an NCRIS facilty and they wanted to focus on the developer community including RSEs. I mentioned the idea of developer advocates which is a US term that is widely used. I wanted to know if anyone had seen this in work within research software?

What is a developer advocate?

In simple terms, a Developer Advocate, also known as a Developer Evangelist or DevRel / dRel (Developer Relations) Advocate, is a technical advocate or ambassador for a company, organization, or platform that develops software, APIs, or other technology products.
Their primary role is to bridge the gap between developers and the company or organization they represent.

From Hushen Savani - [What the heck is a Developer Advocate? And how to become one?](https://www.linkedin.com/pulse/what-heck-developer-advocate-how-become-one-hushen-savani/)

**Comment by rowlandm** [comment]
- Grey space from develop to configuration of platforms that could be fedback from devs to platform devs.
- Born of a need in commercial software organisations that are so big. One extreme of the maturity model.
- A tax on actual developers doing what they do - service model doesn't always work in the academy
- Computing is the water fountain at the end of the hall - you need to be hydrated to do good research.

**Comment by rowlandm** [comment]
- Campus champions program with XCEDE and NSF - contribute to an FTE (like a fellowship / champion) and bring people on board
- Separate: History of RSE would be really interesting in each country and also in domains like Bioinformatics
- When it comes to layoffs this role is targeted

**Reply by jduckles** [reply]
More on the [US champions program](https://campuschampions.cyberinfrastructure.org)...it is basically a cohort of people in the usa who get their management / leadership to sign the letter at the top of the page here: https://campuschampions.cyberinfrastructure.org/form/join-campus-champions

**Comment by rowlandm** [comment]
- Good for people who likes to speak and technical
- In QLD and NSW like QCIF and Intersect they had folk embedded in the universities and directed work back to Intersect

**Comment by rowlandm** [comment]
- There needs to be funding to cover this.

**Comment by rowlandm** [comment]
- Lots of technical debt when funding is cyclic.

**Comment by rowlandm** [comment]
- raising profile for fellows and share information
- all the people who treid to use the product and it was too difificult and stopped - these are the people you want to talk to.

**Comment by rowlandm** [comment]
- programs to make other states more competitive in the states.
- We need to make sure everyone got access in the state - [Oklahoma cyber](https://campuschampions.cyberinfrastructure.org/) - acccess model to get access even if it is paid
- professional excuse to work with each other. You have to do it yourself but it created a network of professionals.

**Reply by jduckles** [reply]
This is the correct link for the shared Oklahoma Cyberinfrastructure initiative. https://www.oneocii.okepscor.org/

**Comment by rowlandm** [comment]
- If you get national funding in the Netherlands you get RSE time in the budget at the national center.

**Comment by rowlandm** [comment]
- Environments with tools and platforms and how to support?

**Comment by rowlandm** [comment]
- What are the differences between Astronomy and Bioinformatics around interoperability.

**Reply by rowlandm** [reply]
Question around commercial instruments vs open instruments!

**Comment by rowlandm** [comment]
- Awareness and Ease of use for providing platforms to make it more sustainable
- Jars of specimens that are kept in perpetuity are paid for as they are seen as valuable.

**Comment by rowlandm** [comment]
- Big green button - few minutes later you have infrastructure to do your stuff
- AAF has a small green button
- Effort into making these easier to make it easier to do a big green button

**Comment by rowlandm** [comment]
Minimal computing is an interesting paradigm in the humanities.... https://go-dh.github.io/mincomp/about/
